docs/concepts/password-policy.mdx

```diff
@@ -5,25 +5,25 @@ title: Password policy
 
 Password-based authentication flows are subject to frequent abuse through social engineering, password guessing and phishing
 attacks.  
-Ory Cloud implements measures to provide high security for password-based flows. The Ory Password Policy follows standards by the
-_National Cyber Security Centre_ ([`NCSC`](https://www.ncsc.gov.uk/collection/passwords/updating-your-approach)) and _National
+Ory Network implements measures to provide high security for password-based flows. The Ory Password Policy follows standards by
```

Comment:
We need to use "The Ory Network" in such sentences IMHO.
so "Ory Cloud does xyz" -> "The Ory Network does ...."
"in Ory Cloud" --> "on the Ory Network" 
... it may get tricky in places, LMK if there are issues!

Reply:
That's done, I hope I caught everything.